src/appleseed/renderer/modeling/camera/camera.h

```diff
@@ -157,21 +158,39 @@ class APPLESEED_DLLSYMBOL Camera
 
   protected:
     TransformSequence   m_transform_sequence;
+
+    // Shutter related parameters
     float               m_shutter_open_time;
     float               m_shutter_open_end_time;
     float               m_shutter_close_start_time;
     float               m_shutter_close_time;
     float               m_shutter_open_time_interval;
     float               m_normalized_open_end_time;
-    float               m_normalized_open_end_time_half;
     float               m_normalized_close_start_time;
-    float               m_open_linear_curve_slope;
-    float               m_close_linear_curve_slope;
-    float               m_shutter_pdf_max_height;
     float               m_inverse_cdf_open_point;
     float               m_inverse_cdf_close_point;
     bool                m_motion_blur_enabled;
 
+    // Linear shutter related parameters
+    float m_shutter_curve_linear_open_multiplier;
```

Comment:
Align members vertically.

Reply:
Can I align fields by blocks (with different indentation for each block)?

Reply:
Yes, sure. Aligning members is more of an art form than a science, there's no rule that always work, and there's certainly a level of subjectivity to what looks more pleasing.